index.html

```diff
@@ -0,0 +1,339 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <section class="menu-mobile">
+      <div id="menu-container" class="mobile-menu-container">
+        <div class="bg-img"></div>
+        <button class="close-btn">x</button>
+        <ul class="mobile-menu-ul" id="anchor-tags">
+          <li><a href="#works">Portfolio</a></li>
+          <li><a href="#about-myself">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+    </section>
+    <header class="main-header">
+      <nav>
+        <ul class="header-menu">
+          <li><a href="#" class="info-text">Melkamu</a></li>
+          <li>
+            <a href="#" id="hamburger-btn" class="nav__button"
+              ><i
+                ><img
+                  src="Asset/Icon - Menu.png"
+                  alt="hamburger-icon-menu"
+                  class="info-image"
+                  id="hamburger-btnicon" /></i
+            ></a>
+          </li>
+          <li class="desktop-menu-container">
+            <ul class="menu-lists">
+              <li class="item"><a href="#portfolio">Portfolio</a></li>
+              <li class="item"><a href="#about">About</a></li>
+              <li class="item"><a href="#contact">Contact</a></li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-content">
+      <section class="headlines">
+        <div class="container">
+          <div class="about" id="about-info">
+            <h1 class="intro">I’m Melkamu.<br />Glad to see you!</h1>
+            <p>
+              I'm a software developer! I can help you build a product, feature,
+              or website. Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <h3 class="connect">LET’S CONNECT</h3>
+          <ul class="social">
+            <li>
+              <a href=""><img src="Asset/vector1git.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/LinkidIn.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/angellist.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/Twitter.png" alt="" /></a>
+            </li>
```

Comment:
- [ ] Kindly fix the designs deficiency in the main page part.

- Figma design
<img width="379" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/b72ae7fa-9c14-47c6-b478-57d4b663d133">

- Your design
<img width="463" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/5b0f0914-5d72-4965-b6b6-549c46b4e014">

- Fix the element spacing wherever there is an arrow.
- The circle at the left-bottom corner indicates that the border in your design, should be more curvy.
- The circle around the nav items, indicates that the text should be in cap case. And the styling should match that of the Figma design.
- OPTIONAL - Your background image resolution is a bit low.